### ISSUE TITLE
Error parsing example from keepachangelog.com

### ISSUE THREAD

**Issue by tobli** [contentful-labs/keepachangelog#10]
Hi,
First of all, thanks for building this and releasing it as open source. When trying to read the example file from http://keepachangelog.com/CHANGELOG.md, parsing fails with the following error:
TypeError: undefined is not a function

To try it out, I added pasted the file contents into test/fixtures/keepachangelogexample.md, and added the following test case. 

``` javascript
  it('parses example from keepachangelog.com', function() {
    var source = readFixture('keepachangelogexample')
    var changelog = parse(source)
    expect(changelog.build()).to.equal(source)
  });
```

Best regards,
/Tobias

**Comment by geigerzaehler** [comment]
Hi Tobias. The library currentl does not support link references. I opened a PR (#11) that is WIP, but will add this eventually. I will also add the Keepachangelog Change Log to the fixtures to enforce compatibility.

**Comment by tobli** [comment]
Great! Looking forward to getting this released, but will avoid link references until then.

**Comment by geigerzaehler** [comment]
Should be fixed in `v0.6.0`.
